File: src/main/java/com/weeth/WeethApplication.java
package com.weeth;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
import org.springframework.scheduling.annotation.EnableScheduling;
import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;

@EnableScheduling
@EnableJpaAuditing
@EnableWebSecurity
@SpringBootApplication
@ConfigurationPropertiesScan
public class WeethApplication {

    public static void main(String[] args) {
        SpringApplication.run(WeethApplication.class, args);
    }

}

File: src/main/kotlin/com/weeth/WeethApplication.kt
package com.weeth

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaAuditing
import org.springframework.scheduling.annotation.EnableAsync
import org.springframework.scheduling.annotation.EnableScheduling
import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity

@EnableAsync
@EnableScheduling
@EnableJpaAuditing
@EnableWebSecurity
@SpringBootApplication
@ConfigurationPropertiesScan
class WeethApplication

fun main(args: Array<String>) {
    runApplication<WeethApplication>(*args)
}
